next/api/resources/retired.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, request, send_file
```

Comment:
Can we put this file in the experiment list file? It fundamentally has to with the experiment list and is less spaghetti code.